pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -2776,6 +2776,20 @@ in
 
   rocks-nvim = neovimUtils.buildNeovimPlugin {
     luaAttr = luaPackages.rocks-nvim;
+
+    # luaInterpreter = config.programs.neovim.package.lua;
+
+    #       "${luaInterpreter.pkgs.luarocks_bootstrap}/bin/luarocks"
+    # should I add a luarocksConfig ?
+    # ["rocks_trees"] = {
+    #   {
+    #     ["name"] = "rocks.nvim",
+    #     ["root"] = "/home/teto/.local/share/nvim/rocks"
+    #   },
```

Comment:
Was this supposed to be a TODO of some sort?

Reply:
cargoculted some of my config which doesn't seem relevant anymore. Anyhow I removed it